docker-compose.override.yaml

```diff
@@ -8,7 +8,7 @@ services:
     image: fcp-fd-data-development
     container_name: fcp-fd-data-development
     ports:
-      - "3000:3000"
+      - "3004:3004"
       - "9229:9229"
```

Comment:
Might be worth just updating the debug port as well to `9004:9229`?

Reply:
good spot thanks